hoomd/state.py

```diff
@@ -257,17 +293,20 @@ def update_group_dof(self):
 
     @property
     def cpu_local_snapshot(self):
-        """hoomd.data.LocalSnapshot: Directly expose HOOMD data buffers
-        on the CPU.
+        """hoomd.data.LocalSnapshot: Expose simulation data on the CPU.
 
         Provides access directly to the system state's particle, bond, angle,
         dihedral, improper, constaint, and pair data through a context manager.
-        The `hoomd.data.LocalSnapshot` object is only usable within a
-        context manager (i.e. ``with sim.state.cpu_local_snapshot as data:``)
-        The interface is similar to that of the `hoomd.Snapshot`. Data is local
-        to a given MPI rank. The returned arrays are `hoomd.array.HOOMDArray`
-        objects. Through this interface zero-copy access is available (access is
-        guarenteed to be zero-copy when running on CPU, may be zero copy if
+        Data in `State.cpu_local_snapshot` is MPI rank local, and the
+        `hoomd.data.LocalSnapshot` object is only usable within a context
+        manager (i.e. ``with sim.state.cpu_local_snapshot as data:``).  Attempts
+        to assess data outside the context manager will result in errors.  The
```

Comment:
Sometimes I've noticed that vim inserts two spaces after periods when wrapping. Not sure if that's how this happened.
```suggestion
        manager (i.e. ``with sim.state.cpu_local_snapshot as data:``). Attempts
        to assess data outside the context manager will result in errors. The
```